src/Symfony/Component/Asset/Preload/PreloadManager.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Asset\Preload;
+
+/**
+ * Manages preload HTTP headers.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class PreloadManager implements PreloadManagerInterface
```

Comment:
would forbid decoration for no reason imho - see comments on the interface: building a profiler panel could require decorating this class